src/bitmessageqt/__init__.py

```diff
@@ -1764,50 +1937,80 @@ def updateSentItemStatusByAckdata(self, ackdata, textToDisplay):
                         sent.item(i, 3).setText(textToDisplay)
 
     def removeInboxRowByMsgid(self, msgid):  # msgid and inventoryHash are the same thing
+        """TBC"""
+
         for inbox in ([
-            self.ui.tableWidgetInbox,
-            self.ui.tableWidgetInboxSubscriptions,
-            self.ui.tableWidgetInboxChans]):
+                self.ui.tableWidgetInbox,
+                self.ui.tableWidgetInboxSubscriptions,
+                self.ui.tableWidgetInboxChans]):
             for i in range(inbox.rowCount()):
                 if msgid == str(inbox.item(i, 3).data(QtCore.Qt.UserRole).toPyObject()):
                     self.updateStatusBar(
                         _translate("MainWindow", "Message trashed"))
                     treeWidget = self.widgetConvert(inbox)
-                    self.propagateUnreadCount(inbox.item(i, 1 if inbox.item(i, 1).type == AccountMixin.SUBSCRIPTION else 0).data(QtCore.Qt.UserRole), self.getCurrentFolder(treeWidget), treeWidget, 0)
+                    self.propagateUnreadCount(
+                        inbox.item(
+                            i,
+                            1 if inbox.item(
+                                i,
+                                1).type == AccountMixin.SUBSCRIPTION else 0).data(
+                                    QtCore.Qt.UserRole),
+                        self.getCurrentFolder(treeWidget),
+                        treeWidget,
+                        0)
                     inbox.removeRow(i)
                     break
 
     def newVersionAvailable(self, version):
+        """TBC"""
+
         self.notifiedNewVersion = ".".join(str(n) for n in version)
-        self.updateStatusBar(_translate(
-            "MainWindow",
-            "New version of PyBitmessage is available: %1. Download it"
-            " from https://github.com/Bitmessage/PyBitmessage/releases/latest"
+        self.updateStatusBar(
+            _translate(
+                "MainWindow",
+                "New version of PyBitmessage is available: %1. Download it"
+                " from https://github.com/Bitmessage/PyBitmessage/releases/latest"
```

Comment:
Why? It was formatted before.

Reply:
pylint was complaining about indentation, even though pycodestyle and flake8 were happy.